course/src/main/java/org/openedx/course/presentation/unit/video/VideoFullScreenFragment.kt

```diff
@@ -69,6 +71,9 @@ class VideoFullScreenFragment : Fragment(R.layout.fragment_video_full_screen) {
         if (viewModel.isPlaying == null) {
             viewModel.isPlaying = requireArguments().getBoolean(ARG_IS_PLAYING)
         }
+        viewModel.transcripts = stringToObject<Map<String, String>>(
+            requireArguments().getString(ARG_TRANSCRIPTS, "")
+        ) ?: emptyMap()
```

Comment:
If you intended to pass it to the ViewModel constructor, it may not be feasible to pass a map, and passing it as a string might not be ideal. Please let me know if you had something else in mind.

Reply:
Let's proceed with the current approach.